specs/Compatibility/RuntimeCompatibilityOptions.md

```diff
@@ -0,0 +1,226 @@
+when updating the app to 1.9, you may choose to simply update `PatchLevel1` to 1.9.3 and leave
+`PatchLevel2` as 1.8.2.
+
+### Specifying RuntimeCompatibilityOptions in the app's project file
+
+You can also specify the patch level and disabled changes in the app's project file instead of
+directly calling the RuntimeCompatibilityOptions APIs. This approach has the advantage of ensuring the
+options are applied early at the proper timing. Here's an example of how to specify the patch
+level and disabled changes in the project file (such as `.csproj` or `.vcxproj`):
+
+```xml
+  <PropertyGroup>
+    <WindowsAppSDKRuntimePatchLevel>1.7.3</WindowsAppSDKRuntimePatchLevel>
+    <WindowsAppSDKRuntimePatchLevel2>1.8.2</WindowsAppSDKRuntimePatchLevel2>
+    <WindowsAppSDKDisabledChanges>SampleApiCrashFix, OtherSampleApiCrashFix</WindowsAppSDKDisabledChanges>
+  </PropertyGroup>
+```
+
+The `WindowsAppSDKDisabledChanges` property is a comma-separated list of `RuntimeCompatibilityChange`
+values to disable.
+
+### Behavior with no PatchLevel specified
+
+If no `PatchLevel1` or `PatchLevel2` is specified, or if neither value matches the major.minor
+version of the runtime being used, then the runtime will use the latest patch level. In other
+words, the runtime will run with all servicing changes enabled (just like how Windows App SDK
+worked before this API existed).
+
+## RuntimeCompatibilityOptions.PatchLevel1
+
+Optional patch level to use if the runtime version matches the major.minor version. If the runtime
+version does not match the specified major.minor version, this value is ignored.
+
+Instead of directly using this API, this patch level could be specified in the app's project file,
+like this:
+
+```xml
+  <PropertyGroup>
+    <WindowsAppSDKRuntimePatchLevel>1.7.3</WindowsAppSDKRuntimePatchLevel>
+  </PropertyGroup>
+```
+
+If your app is not in the process of transitioning to a new version of the Windows App SDK, you
+may only set this one patch level.
+
+## RuntimeCompatibilityOptions.PatchLevel2
+
+Optional patch level to use if the runtime version matches the major.minor version. If the runtime
+version does not match the specified major.minor version, this value is ignored.
+
+This property enables setting a second patch level to help apps transitioning to a new version of
+the Windows App SDK. This is a convenience to allow the patch levels for both the old and new
+version to be specified during the transition. Apps not in the process of transitioning should
+only set the one patch level they want to use.
+
+Setting both patch levels for the same major.minor version, such as 1.7.3 and 1.7.4, is not allowed
+and will generate an error during `Apply()`.
+
+Instead of directly using this API, this patch level could be specified in the app's project file,
+like this:
+
+```xml
+  <PropertyGroup>
+    <WindowsAppSDKRuntimePatchLevel2>1.8.2</WindowsAppSDKRuntimePatchLevel2>
```

Comment:
Why not
```xml
<WindowsAppSDKRuntimePatchLevels>1.7.1,1.8.2</WindowsAppSDKRuntimePatchLevels>
```
?

Reply:
The intent is to stay close to the API, which could then make the question the same as the question for line 216, where we want to limit to just two versions.